kpops/cli/main.py

```diff
@@ -25,9 +25,7 @@
 
 LOG_DIVIDER = "#" * 100
 
-app = typer.Typer(
-    pretty_exceptions_show_locals=False,
-)
+app = typer.Typer(pretty_exceptions_show_locals=False, pretty_exceptions_enable=False)
```

Comment:
is the first parameter still necessary if _pretty exceptions_ are disabled entirely anyways?